### ISSUE TITLE
Load the Model and Dataset

### ISSUE THREAD

**Issue by abondre98** [abondre98/Adaptive-Global-LCA-Advisor-with-Dynamic-EF-Retrieval#63]
○	Have your LoRA model (initialized with the configuration from the previous step) and training/validation datasets ready.
Example (using a custom script):
 # Pseudocode
train_loader = DataLoader(train_dataset, batch_size=8, shuffle=True)
val_loader = DataLoader(val_dataset, batch_size=8, shuffle=False)

for epoch in range(num_epochs):
    for batch in train_loader:
        outputs = lora_model(**batch)
        loss = outputs.loss
        loss.backward()
        optimizer.step()
        optimizer.zero_grad()
        # Possibly log loss here
